### PR TITLE
Changed to allow sending XML as file.

### PR DESCRIPTION
**Pull request by tank104**

### CODE REVIEW

**Comment by tank104** [comment]
@abuzuhri  just wanted to check if there was a specific reason you used ASCII for byte conversion for XML? I think should be UTF8/UTF16 so it has support of all languages.

**Comment by abuzuhri** [comment]
> @abuzuhri just wanted to check if there was a specific reason you used ASCII for byte conversion for XML? I think should be UTF8/UTF16 so it has support of all languages.

Hi @tank104 

No reason from my side , please mention the code to review

**Comment by tank104** [comment]
This is the code to review

File: Source/FikaAmazonAPI/Services/FeedService.cs [hunk 1]
```diff
@@ -171,30 +167,14 @@ public async Task<CreateFeedDocumentResult> CreateFeedDocumentAsync(ContentType
...
         /// <param name="feedType"></param>
         /// <returns></returns>
-        public async Task<string> SubmitFeedAsync(string XmlContentOrFilePath, FeedType feedType, List<string> marketPlaceIds = null, FeedOptions feedOptions = null, ContentType contentType = ContentType.XML)
+        public async Task<string> SubmitFeedAsync(string feedContentOrFilePath, FeedType feedType, List<string> marketPlaceIds = null, FeedOptions feedOptions = null, ContentType contentType = ContentType.XML)
         {
-
             //We are creating Feed Document
             var feedCreate = CreateFeedDocument(contentType);
 
             //Uploading encoded invoice file
```
Comment (abuzuhri) [inline Source/FikaAmazonAPI/Services/FeedService.cs:175]: we cant keep it only file , many others developer use this library and pass it as text XML , we need to find another solution

**Review by abuzuhri — APPROVED** [review]

**Pull request merged**